static/app/components/events/breadcrumbs/breadcrumbsTimeline.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  BREADCRUMB_TIMESTAMP_PLACEHOLDER,
+  BreadcrumbTimeDisplay,
+  getBreadcrumbColorConfig,
+  getBreadcrumbIcon,
+  getBreadcrumbTitle,
+} from 'sentry/components/events/breadcrumbs/utils';
+import {BreadcrumbSort} from 'sentry/components/events/interfaces/breadcrumbs';
+import {convertCrumbType} from 'sentry/components/events/interfaces/breadcrumbs/utils';
+import {StructuredData} from 'sentry/components/structuredEventData';
+import Timeline from 'sentry/components/timeline';
+import type {RawCrumb} from 'sentry/types/breadcrumbs';
+import {defined} from 'sentry/utils';
+
+interface BreadcrumbsTimelineProps {
+  breadcrumbs: RawCrumb[];
+  meta?: Record<string, any>;
+  sort?: BreadcrumbSort;
+  timeDisplay?: BreadcrumbTimeDisplay;
+  virtualCrumbIndex?: number;
+}
+export default function BreadcrumbsTimeline({
+  breadcrumbs,
+  virtualCrumbIndex,
+  sort = BreadcrumbSort.NEWEST,
+  timeDisplay = BreadcrumbTimeDisplay.RELATIVE,
+  meta = {},
+}: BreadcrumbsTimelineProps) {
+  if (!breadcrumbs.length) {
+    return null;
+  }
+
+  const startTimestamp =
+    timeDisplay === BreadcrumbTimeDisplay.RELATIVE
+      ? breadcrumbs[breadcrumbs.length - 1].timestamp
```

Comment:
nit - could use the newer syntax 
```suggestion
      ? breadcrumbs.at(-1).timestamp
```

Reply:
TIL, didn't know this was in JS